prometheus_scrape_cache.go

```
@@ -27,16 +29,27 @@ import (
 // put in X-header for exposition format
 
 func main() {
+	var (
+		showVersion = flag.Bool("version", false, "Print version information.")
+		listenAddress = flag.String("web.listen-address", ":8080", "Address to listen on for web interface and telemetry.")
+		metricsPath   = flag.String("web.telemetry-path", "/prometheus_scrape_cache/metrics", "Path under which to expose metrics.")
+		baseUrl = flag.String("base.url", "http://localhost:8080/metrics", "Base URL to scrape from")
```

I would tend to simply fail when this options is not set.